src/index.js

```diff
@@ -61,7 +62,7 @@ export default function (CustomElement, opts) {
       });
     }
     render() {
-      return React.createElement(tagName, null, this.props.children);
+      return React.createElement(tagName, { style: this.props.style }, this.props.children);
```

Comment:
Would be transpiled to a call that contains `Array.from`. One of our mobile browsers we need to support doesnt know that yet if I remember correctly.